src/database/migrations/20190819160153_create_tables.js

```diff
@@ -0,0 +1,63 @@
+exports.up = kn =>
+  kn.schema
+    .createTable("projects", tb => {
+      tb.increments();
+
+      tb.string("name")
+        .notNullable()
+        .unique();
+
+      tb.string("description");
+
+      tb.boolean("completed").defaultTo(false);
+    })
```

Comment:
I would test to see what this does when you enter something and set the completed to true....you may not want to set the default here....but in your model.